### DISCUSSION TITLE
blog/2024/02/25/movemenoreg-vbs-analysis/

### DISCUSSION THREAD

**Discussion by giscus** [L0serQianXia/blog#8] [Announcements]
# blog/2024/02/25/movemenoreg-vbs-analysis/

https://l0serqianxia.github.io/blog/2024/02/25/movemenoreg-vbs-analysis/

**Comment by Cznorth** [comment]
感谢大佬的分析报告，我们学校实验室也中了这个病毒https://www.cznorth.cn/index.php/2024/04/27/%e5%ad%a6%e6%a0%a1%e5%ae%9e%e9%aa%8c%e5%ae%a4u%e7%9b%98%e7%97%85%e6%af%92%e6%9f%a5%e6%9d%80%e8%ae%b0%e5%bd%95/

**Reply by L0serQianXia** [reply]
关于你的文章里提到没有显示U盘里隐藏文件的问题，还是希望确认一下，文件夹选项中是否有取消勾选“隐藏受保护的操作系统文件(推荐)”？
![图片](https://l0serqianxia.github.io/blog/2024/02/25/movemenoreg-vbs-analysis/image-20240225113003987.png)
原文没对图片上加醒目的标注，目前已经修改了
